docs/releases/release-support-policy.md

```diff
@@ -0,0 +1,83 @@
+# Release Support Policy
+
+This page describes Scalar's support policy for major and minor version releases of ScalarDB.
+
+## Terms and definitions
+
+- **Maintenance Support:** Scalar will provide product updates, including code fixes, documentation, and technical support through its [support portal](https://support.scalar-labs.com/) to customers with a commercial license, until the date specified.
+- **Assistance Support:** Scalar will continue to provide technical support through its [support portal](https://support.scalar-labs.com/) to customers with a commercial license until the date specified.
```

Comment:
@josh-wong  Is there any good way to distinguish our technical support between Maintenance Support and Assitance Support? We can provide the limited technical support only in Assistance Support (mainly for QA support about upgrade and how-to, since we cannot provide the detailed troubleshooting support with the code fixed.)

Reply:
@NatsukiMiyamoto I see what you mean. What do you think about changing this sentence as follows?:

```suggestion
- **Assistance Support:** Scalar will provide limited technical support for non-code-related questions in the form of FAQs and inquiries through its [support portal](https://support.scalar-labs.com/) to customers with a commercial license until the date specified.
```

Reply:
LGTM. Thank you!